tests/integration/it_owner_privilages_test.py

```diff
@@ -16,6 +16,7 @@ def test_cant_access_unknown_resource(app):
 
 
 def test_cant_access_his_profile(app):
+    flask.g.user = None
```

Comment:
@krzyk is it necessary to specify ``None``? Isn't that the default value?

Reply:
@amihaiemil it is not, not sure where it is set, but without it the test breaks